R/init.R

```diff
@@ -97,14 +98,12 @@ init <- function(data,
                  title = build_app_title(),
```

Comment:
Deprecated function can't be a default. It will throw each time when `title` is not specified

Reply:
Yeah, that's why I changed it to `deprecate_soft()`. This function will still exist, it will just become internal in the next release. Because we use it in two places.

Reply:
Yes, but it will throw deprecation even if one doesn't do anything with `title`. It will for sure come up in tests of init